app/main/presenters/service_presenters.py

```diff
@@ -33,15 +34,35 @@ def __init__(self, service_data, service_questions, lots_by_slug):
             service_data['serviceDescription']
         self.lot = lots_by_slug.get(service_data['lot'])
         self.frameworkName = service_data['frameworkName']
+
+        # Convert internal answer representations (DB values) to human-readable labels
+        translated_service_data = self._convert_values_to_labels(copy.deepcopy(service_data))
+
         # optional attributes directly mapped to service_data values
         for key in [
             ('supplierName', 'supplierName'),
             ('serviceFeatures', 'features'),
             ('serviceBenefits', 'benefits')
         ]:
-            self._add_as_attribute_if_key_exists(key, service_data)
-        self.attributes = self._get_service_attributes(service_data)
-        self.meta = self._get_service_meta(service_data)
+            self._add_as_attribute_if_key_exists(key, translated_service_data)
+
+        self.attributes = self._get_service_attributes(translated_service_data)
+        self.meta = self._get_service_meta(translated_service_data)
+
+    def _convert_values_to_labels(self, service_data):
```

Comment:
Yes. That summary is used by this function to update the raw JSON in service_data, which the Service presenter uses, with their equivalent labels.